pbs_light.go

```diff
@@ -96,7 +96,14 @@ type bidResult struct {
 	bid_list pbs.PBSBidSlice
 }
 
-const DEFAULT_PRICE_GRANULARITY = "med"
+const defaultPriceGranularity = "med"
+
+const hbCreativeLoadMethodDemandSDK = "demand_sdk"
+const hbCreativeLoadMethodHTML = "html"
```

Comment:
Could you leave some comments describing the purpose of these two?

The info you gave me about `demand_sdk` being used to target bidders who insist on control over the webview,  and `html` being used as the default, seem pretty important.

I guess all I'm hoping for is... if we stumble on this code in 3 years and wonder "do we still need this?" We'll need to know what purpose it serves in the first place.